### DISCUSSION TITLE
Pickable children

### DISCUSSION THREAD

**Discussion by eqdiag** [aevyrie/bevy_mod_picking#152] [Q&A]
Hey, I'm trying to create a pickable entity that is a child of another pickable entity. I am making them both pickable by adding the appropriate components via `insert_bundle(PickableBundle::default())`. When the two entities don't have a parent-child relationship, I'm able to highlight both entities, but once one of them becomes a child of the other, the child entity isn't highlighted. Any ways to make highlighting the child entity work?

**Comment by aevyrie** [comment]
This is a bug. Please create an issue for this, with a minimum example so I can reproduce.
